### DISCUSSION TITLE
Import i3D for settings only

### DISCUSSION THREAD

**Discussion by Mike1977** [StjerneIdioten/I3D-Blender-Addon#149] [Ideas & Feature Requests]
I know its not possible to import an i3D since the shapes are locked in an encryptet format. But all the settings like material, clip distance, shaders, collision mask ect, are stored in the readable portion of the i3d. It would be sooo helpful if i could just import all the settings from an i3d instead of opening the i3d in an editor and transfering all the settings into blender by hand.

**Comment by StjerneIdioten** [comment]
That is true and will most probably be a feature of the next version. It just wasn't ever a priority to import anything for FS19 mods since some things had to be converted regardless from previous mods. But I guess now with FS22, people want to open their old I3D's or convert some existing mods :-P

**Comment by Mike1977** [comment]
First of all, sorry for opening this as an issue. I wasn't sure where to post this, and so i went for both locations ;)
And even though im mapping since FS11, I still consider myself as a beginner. Thats why i still work on extending already existing maps, mainly because im not nearly creative enough to come up with a new map from scratch. I converted one of the maps that came with FS22 into a 4x Map, and now i have to fill space the size of 3 maps with content. And here comes the part where i have to import streets, bridges, pathways ect and change it, so it fits ecactly the look of the original map and im still flexible enough to build the Map without using streetpacks and such. 
Its a very tedious task to import all the settings for every object, and i think for people like me sucha feature would be a huge timesaver....

**Reply by StjerneIdioten** [reply]
Ah yes, the exporter has never really been developed with mapping in mind :-P Since most people wanted something for just vehicles and the workflows there and I never got to the point where I could start on making mapping easier. But maybe in the future ;-)

**Reply by Mike1977** [reply]
So, since you said this will be probably a thing in the next version, my question of course is now what a timeframe we are talking about. 
Is it a few weeks, months, near the end of the year?

**Reply by StjerneIdioten** [reply]
I hope it will be in the span of months, but it really depends on how much time I have on my hands 🤷🏻

**Reply by Mike1977** [reply]
Ah, what a bummer... that means i have to keep doing all the work by hand, and when im done with the project, you will probably release the timesaver tool  :D

**Reply by kbrandwijk** [reply]
That's why time machines are such a great invention. You can wait for something to be completed, use it, and go back in time to submit your map :)
